Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM ruby:2.6.6
+
+# install java & set path
+RUN apt-get update && \
+    apt-get -y install -y sudo
+
+RUN apt-get install -y openjdk-11-jdk
+RUN sudo update-alternatives --config java
+
+ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk-amd64/
+RUN export JAVA_HOME
+RUN javac -version
+
+# install ant
+RUN apt update
+RUN apt -y install ant
+RUN ant -version
+
+# install antlr
+WORKDIR /usr/local/lib
+RUN mkdir -p /usr/java/lib
```

Comment:
Requires root permission.  Do you need to make that clear in your documentation that the install requires root permissions?

Reply:
I think this should not be an issue since I did not define a non-root user.